src/About/About.js

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import AboutImage from '../../public/images/VirtuousCycle.svg';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    margin: theme.spacing(4)
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+  sectionHeader: {
+    // margin: theme.spacing(4),
+    padding: theme.spacing(2),
+    textAlign: 'center',
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(15),
+    flexBasis: '33.33%',
+    flexShrink: 0,
+  },
+  secondaryHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    color: theme.palette.text.secondary,
+  },
+});
+
+class About extends React.Component {
+
+  render() {
+    const { classes } = this.props;
+
+    return(
+      <div className={classes.root}>
+        <div className={classes.heroContent}>
+          <Typography variant="h1" align="center" color="textPrimary" gutterBottom>
+            About DeepCell
```

Comment:
Can we remove this header? It is a bit redundant and doesn't fit with the aesthetic of the rest of the page.

Reply:
Sure, I've mocked that up (haven't pushed the commit yet). How does it look?